goal_src/jak2/engine/math/transformq-h.gc

```diff
@@ -73,7 +73,7 @@
     (global-y-angle-to-point (_type_ vector) float 23)
     (relative-y-angle-to-point (_type_ vector) float 24)
     (roll-relative-to-gravity (_type_) float 25)
-    (set-and-limit-velocity (_type_ int vector float) trsqv 26)
+    (set-and-limit-velocity (_type_ int vector float) trsqv :behavior process 26)
```

Comment:
If this is just looking at process clocks, I want to eventually turn this into a macro that doesn't require making things a behavior.

Reply:
It is yeah, ill make an issue